VisualStudio.gitignore

```diff
@@ -22,24 +24,37 @@ bld/
 [Bb]in/
 [Oo]bj/
 
-# Visual Studio 2015 cache/options directory
-.vs/
-
-# MSTest test Results
+# MSTest
+# a test framework from Microsoft
+# remove: if you are not using the MSTest runner to run your tests
 [Tt]est[Rr]esult*/
 [Bb]uild[Ll]og.*
 
-# NUNIT
+# NUnit
+# an open source test framework for .NET
+# site: http://www.nunit.org/
+# remove: if you are not using the NUnit test runner to run your tests
```

Comment:
This is an example of gathering up rules related to a specific tool - . It feels a bit promotion-y, but also feels like a good way for users to explore the ecosystem better.